### PR TITLE
add php stan to tests

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -55,12 +70,14 @@ cache:
...
   global:
     - TASK_TESTS=1
+    - TASK_STAN=0
     - TASK_DOCS=0
     - PIMCORE_ENVIRONMENT=test
     - PIMCORE_TEST=1
     - PIMCORE_TEST_URL=http://pimcore-test.dev
     - PIMCORE_TEST_DB_DSN="mysql://root@localhost/pimcore_test"
     - PIMCORE_TEST_CACHE_REDIS_DATABASE=1
+    - PIMCORE_TEST_CACHE_REDIS_DATABASE=1
```
Comment (jdreesen) [inline .travis.yml]: This is the same as the line above.

Comment (dpfaffenbauer) [reply .travis.yml]: yes, but that was the case before as well. I will wait for #4435 and continue working on this

Comment (dpfaffenbauer) [reply .travis.yml]: it was not, sorry. But this is WIP anyway :)

**Comment by brusch** [comment]
#4435 is merged now 🎉
Can you resolve the resulting conflict in this PR and remove the duplicate line as @jdreesen noticed? 
Thanks again!

**Comment by dpfaffenbauer** [comment]
Will do.

Also have it make Stan optional for now. It needs a lot of work to fix the issues.

**Comment by dpfaffenbauer** [comment]
@brusch this is finished now. Next is to make stan tests actually work, which is the biggest part.

**Comment by dpfaffenbauer** [comment]
sorry, no. not done yet. there is a different error

**Comment by dpfaffenbauer** [comment]
@brusch now it is finished!

**Comment by dpfaffenbauer** [comment]
stan result:

```
[ERROR] Found 1445 errors
```

File: composer.json [hunk 2]
```diff
@@ -80,7 +80,7 @@
     "symfony-cmf/routing-bundle": "^2.0",
     "symfony/monolog-bundle": "^3.1.0",
     "symfony/swiftmailer-bundle": "^3.2.2",
-    "symfony/symfony": "^3.4.17 || ^4.0",
+    "symfony/symfony": "3.4",
```
Comment (jdreesen) [inline composer.json]: I think this change should be reverted.

Comment (dpfaffenbauer) [reply composer.json]: oh boy... totally, reverted it!

**Comment by dpfaffenbauer** [comment]
@brusch also rebased to latest master, let's see if it still runs

**Comment by dpfaffenbauer** [comment]
@brusch just rebased to master, when will this be merged?

**Comment by dpfaffenbauer** [comment]
🎉 next step is to remove as many warnings as possible. I will try to create a PR in my free time. But since this is a massive task, some help would be good.

**Comment by dpfaffenbauer** [comment]
just saw you finished level 1, amazing.

What are your plans on the next levels? at least lvl2?

**Pull request merged**